### ISSUE TITLE
Combination of jasmine and spyOn make wallabyjs unstable

### ISSUE THREAD

**Issue by mdreizin** [wallabyjs/public#621]
### Issue description

In some projects I use `jasmine` and `spyOn` and wallabyjs constantly crashes with:

```
Jasmine spies would overwrite the 'and' and 'calls' properties on the object being spied upon
```

If restart wallabyjs then it will work again.

### Wallaby.js configuration file

https://github.com/mdreizin/webpack-cluster/blob/master/wallaby.config.js

### Code editor or IDE name and version

```
WebStorm 2016.2 EAP
Build #WS-162.426.3, built on May 24, 2016
WebStorm EAP User
Expiration date: June 23, 2016
JRE: 1.8.0_76-release-b205 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
```

### OS name and version

OSX

**Comment by ArtemGovorov** [comment]
The issue is fixed in the core v1.0.238.
Note that while running tests in your project I have noticed a couple more issues with tests that may be related to the fact that wallaby.js [reuses node processes by default](https://wallabyjs.com/docs/integration/node.html#node-process-reuse). If you hit those issues, just try setting the `workers: { recycle: true }` setting, it should help.

**Comment by mdreizin** [comment]
@ArtemGovorov Thank you I really appreciate your help.
